src/person-tracking/README.md

```diff
@@ -1,25 +1,43 @@
-# How to build
-```
-cd src/person-tracking
-npm install
-```
+# Intel® RealSense™ technology with JavaScript API for Linux OS
 
-# How to run samples
-```
-cd src/person-tracking
-npm run maketarball
-```
-The above step generates a person-tracking-xxx.tar.gz file.
+`node-person`: Node.js Person Detection/Person Tracking/Person Recognition/Persong Skeleton Tracking/Body Gesture Recognition/Face Tracking algorithm module based on Intel® RealSense™ technology.
```

Comment:
currently, PT supports: 
Person Detection/Person Tracking/Person Skeleton Recognition/Body Gesture Recognition

while, Person Recognition and Face Tracking are the same feature and not supported yet. Is it better to remove those 2 ?